extensions/wikia/TemplateClassification/scripts/TemplateClassificationModal.js

```diff
@@ -163,19 +166,18 @@ function ($, w, mw, loader, nirvana, tracker, throbber, labeling) {
 
 		throbber.remove($throbber);
 
-		// Make sure that focus is in the right place
-		$('#template-classification-' + mw.html.escape($preselectedType.val())).focus();
+		// Make sure that focus is in the right place but scroll the modal window to the top
+
+		$('#template-classification-' + mw.html.escape(preselectedType)).focus();
+		if (preselectedType === 'unknown') {
+			modalInstance.scroll(0);
+		}
 	}
 
 	function processSave(modalInstance) {
-		var newTemplateType = $('#TemplateClassificationEditForm [name="template-classification-types"]:checked').val(),
-			oldTemplateType = '';
```

Comment:
The `track` call in the else block below still uses `oldTemplateType` which is now undefined, so it should be updated there too.